### PR TITLE
Release v4.0.13

### PR DESCRIPTION
**Pull request by devonpis**
sync master with v4.0.13

### CODE REVIEW

File: gulp/common-tasks/scss.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ module.exports = function (gulp, plugins, config, destFolder = 'assets', addSrc)
         browsers: plugins.supportedBrowser,
         cascade: false,
       }))
-      .pipe(sourcemaps.write())
+      .pipe(sourcemaps.write('.'))
```
Comment (duttonw) [inline gulp/common-tasks/scss.js:36]: what does this do?

File: src/docs/examples/formioV2.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<head>
+  <meta http-equiv="refresh" content="2; URL=https://www.bitdegree.org/" />
+</head>
+<body>
+<p>If you are not redirected in 2 seconds, <a href="https://www.bitdegree.org/">click here</a>.</p>
```
Comment (duttonw) [inline src/docs/examples/formioV2.html:5]: Whats this? this does not look right

**Review by duttonw — CHANGES_REQUESTED** [review]
why do we have an example with bitdegree . org in the redirect?

File: gulp/common-tasks/scss.js (hunk 1 above)
Comment (asifamingov) [reply gulp/common-tasks/scss.js:36]: write maps file for debugging in the same folder for local development. Earlier it was outputting in the root, so Chrome could not detect it.

File: src/docs/examples/formioV2.html [hunk 3]
```diff
@@ -0,0 +1,1480 @@
...
+        <script src="/?a=252693" type="text/javascript"></script>
+        <script>
+          // Persistent fix for iPhone/Safari
+          window.onpageshow = function (event) {
+            if (event.persisted) {
+              window.location.reload();
+            }
+          };
+        </script>
+        <style>
```
Comment (devonpis) [inline src/docs/examples/formioV2.html:475]: @asifamingov this style tag has been removed in squiz matrix component

File: src/docs/examples/formioV2.html (hunk 2 above)
Comment (asifamingov) [reply src/docs/examples/formioV2.html]: reverted the content, some test content got committed. Those are just examples for internal testing.

File: src/docs/examples/formioV2.html (hunk 3 above)
Comment (asifamingov) [reply src/docs/examples/formioV2.html]: > @asifamingov this style tag has been removed in squiz matrix component

I deleted those examples of specific forms as those might cause some issues if those are not live and in the examples folder if someone finds those.

File: src/docs/examples/formioV2.html (hunk 2 above)
Comment (asifamingov) [reply src/docs/examples/formioV2.html]: > Whats this? this does not look right

I deleted those examples of specific forms as those might cause some issues if those are not live and in the examples folder if someone finds those.

**Review by duttonw — APPROVED** [review]

**Pull request merged**